src/status_im2/contexts/wallet/send/select_address/view.cljs

```diff
@@ -41,70 +44,155 @@
+        [ens-linked-address local-suggestion]
+        :else nil))))
+
+(defn- local-suggestions-list
+  []
+  (fn []
+    (let [local-suggestion (rf/sub [:wallet/local-suggestions])]
+      [rn/view {:style {:flex 1}}
+       [rn/flat-list
+        {:data                         local-suggestion
+         :content-container-style      {:flex-grow 1}
+         :key-fn                       :id
+         :on-scroll-to-index-failed    identity
+         :keyboard-should-persist-taps :handled
+         :render-fn                    suggestion-component}]])))
 
 (defn- f-view-internal
   []
-  (let [margin-top    (safe-area/get-top)
-        selected-tab  (reagent/atom (:id (first tabs-data)))
-        on-close      #(rf/dispatch [:dismiss-modal :wallet-select-address])
-        on-change-tab #(reset! selected-tab %)]
+  (let [margin-top     (safe-area/get-top)
+        selected-tab   (reagent/atom (:id (first tabs-data)))
+        on-close       (fn []
+                         (rf/dispatch [:wallet/clean-scanned-address])
+                         (rf/dispatch [:wallet/clean-local-suggestions])
+                         (rf/dispatch [:navigate-back]))
+        on-change-tab  #(reset! selected-tab %)
+        input-value    (reagent/atom "")
+        input-focused? (reagent/atom false)]
     (fn []
-      (rn/use-effect (fn [] #(rf/dispatch [:wallet/clean-scanned-address])))
-      [rn/scroll-view
-       {:content-container-style      (style/container margin-top)
-        :keyboard-should-persist-taps :never
-        :scroll-enabled               false}
-       [quo/page-nav
-        {:icon-name           :i/close
-         :on-press            on-close
-         :accessibility-label :top-bar
-         :right-side          :account-switcher
-         :account-switcher    {:customization-color :purple
-                               :on-press            #(js/alert "Not implemented yet")
-                               :state               :default
-                               :emoji               "🍑"}}]
-       [quo/text-combinations
-        {:title                     (i18n/label :t/send-to)
-         :container-style           style/title-container
-         :title-accessibility-label :title-label}]
-       [address-input]
-       [quo/divider-line]
-       [quo/tabs
-        {:style            style/tabs
-         :container-style  style/tabs-content
-         :size             32
-         :default-active   @selected-tab
-         :data             tabs-data
-         :scrollable?      true
-         :scroll-on-press? true
-         :on-change        on-change-tab}]
-       [tab-view @selected-tab]])))
+      (let [valid-ens-or-address? (boolean (rf/sub [:wallet/valid-ens-or-address?]))]
+        [rn/scroll-view
+         {:content-container-style      (style/container margin-top)
+          :keyboard-should-persist-taps :handled
+          :scroll-enabled               false}
+         [quo/page-nav
```

Comment:
beyond this pr and beyond 1.27 priorities. seems we should eventually make this a common connected component for wallet with all of the functionality built in for wallet page nav. 👍 
cc @smohamedjavid, @OmarBasem @ulisesmac @mmilad75

Reply:
created - https://github.com/status-im/status-mobile/issues/17872